### ISSUE TITLE
Testrpc occasionally crashes with "Process 24102 exited with code 7" error

### ISSUE THREAD

**Issue by elenadimitrova** [ConsenSys-archive/ganache-cli-archive#449]
Testrpc occasionally crashes with the error below. The frequency is ~30-40% of all builds which significantly slows down our releases.

```
Error: Process 24102 exited with code 7
    at ChildProcess.<anonymous> (/home/ubuntu/colonyNetwork/node_modules/pshell/lib/index.js:97:19)
    at Object.onceWrapper (events.js:326:30)
    at emitTwo (events.js:135:13)
    at ChildProcess.emit (events.js:224:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:12)
1
```
## Steps to Reproduce (for bugs)
Cannot find the pattern to allow us to reproduce but it seems related to the reasons described in #359

## Your Environment
* Version used: EthereumJS TestRPC v6.0.3 (ganache-core: 2.0.2)
* Node: 9.1.0

**Comment by benjamincburns** [comment]
@elenadimitrova thanks for submitting. It looks like you're running testrpc/ganache-cli as a child process? If you hook into the stderr you should be able to get the log output. Can you do that and post the underlying error here, please?

Out of curiosity, if you need to run ganache-cli as part of your application, why do you fork a child process rather than embedding the server module, or skipping the network entirely and just using the ganache-core Provider directly?

**Comment by elenadimitrova** [comment]
I think this has been resolved as part of fixing #359

**Comment by benjamincburns** [comment]
Sweet! I love it when that happens! Also thanks for closing!
